specification/cognitiveservices/data-plane/ImmersiveReader/stable/v1.0/ImmersiveReader.json

```diff
@@ -0,0 +1,83 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Immersive Reader API v1.0",
+		"version": "1.0"
+	},
+	"x-ms-parameterized-host": {
+		"hostTemplate": "{Endpoint}/immersivereader/v1.0",
+		"useSchemePrefix": false,
+		"parameters": [{
+				"$ref": "#/parameters/Endpoint"
+			}
+		]
+	},
+	"securityDefinitions": {
+		"apiKeyHeader": {
+			"type": "apiKey",
+			"name": "Ocp-Apim-Subscription-Key",
+			"in": "header"
+		}
+	},
+	"security": [{
+			"apiKeyHeader": []
+		}
+	],
+	"paths": {
+		"/getcontentmodelforreader": {
+			"post": {
+				"description": "Creates an Immersive Reader session",
+				"operationId": "post-getcontentmodelforreader",
```

Comment:
OperationId, should follow the Noun_Verb pattern and should be in PascalCase like this "ContentModelForReader_Get".
By following the above recommendation, the code generator will generate sdk methods in the following format
```js
new ImmerSiveReaderClient().contentModelForReader.get();
```

Reply:
Not looking to create an SDK for this. It is an internal-only API. 

Looking at other cognitive services, their OperationId seems to follow the format 'CognitveServiceName_EndpointName'.

So in this case, that would result in 'ImmersiveReader_GetContentModelForReader'